zkevm-circuits/src/copy_circuit.rs

```diff
@@ -805,20 +809,35 @@ mod tests {
         operation::RWCounter,
     };
     use eth_types::{bytecode, geth_types::GethData, Word};
+    use mock::test_ctx::helpers::account_0_code_account_1_no_code;
     use mock::TestContext;
     use rand::{prelude::SliceRandom, Rng};
 
+    use crate::evm_circuit::test::rand_bytes;
     use crate::evm_circuit::witness::block_convert;
 
     fn gen_calldatacopy_data() -> CircuitInputBuilder {
+        let length: usize = rand::thread_rng().gen_range(0x00..0x0fff);
```

Comment:
I think it would be useful to add `dbg!(length)` after this, so that in case the test fails, we can reproduce it.

Reply:
I'd use a known and easy to identify number such as `0xcafe`.
See: https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/641

On that way, will be less required to debug stuff because the random values are known beforehand and the tests are easier to reproduce